pkg/server/flags.go

```diff
@@ -56,12 +60,27 @@ func (flags *InstallFlags) SetFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&flags.GpuDevices, "gpu-devices", "", "GPU devices to be used (e.g. 1 or 0,1,2 or all)")
 	cmd.Flags().UintVar(&flags.Gpus, "gpus", 0, "Number of GPUs to be used")
 	cmd.Flags().BoolVar(&flags.UseCpu, "cpu", false, "Use CPU for training and evaluating")
-	cmd.Flags().StringVar(&flags.DatasetDirectory, "dataset-dir", "", "Dataset directory maps the user's local directory to the container's directory, enabling access to code integration for training and evaluation")
+	cmd.Flags().StringArrayVarP(&flags.DatasetVolumes, "dataset-volume", "v", []string{}, "Dataset volume maps the user's local directory to the container's directory, enabling access to code integration for training and evaluation")
```

Comment:
Dataset volume maps the user's local directory to the container's directory, enabling Tensorleap platform access to the dataset host folder.